shared/charts/gpii-flushtokens/templates/cronjob.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
+  name: {{ template "flushtokens.name" . }}
+spec:
+  schedule: "*/2 * * * *"
+  concurrencyPolicy: Allow
+  startingDeadlineSeconds: 200
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: {{ template "flushtokens.name" . }}
+        spec:
+          containers:
+          - name: gpii-flushtokens
+            image: "{{ .Values.image.repository }}@{{ .Values.image.checksum }}"
+            command: [ '/app/scripts/dockerDeleteExpiredAccessTokens.sh' ]
+            env:
+            - name: GPII_COUCHDB_URL
+              value: '{{ .Values.couchdbUrl }}'
+            - name: DELETE_ALL_FLAG
```

Comment:
I would call this `DELETE_ALL` -- don't see a lot of Hungarian notation in environment variables ;).

Reply:
Sure.